src/main/java/screens/LoginUI.java

```diff
@@ -35,62 +35,59 @@ public static void main(String[] args) {
 
     public void start(Stage primaryStage){
```

Comment:
Maybe add Javadoc to this method?

Reply:
A few comments within the method itself might make things clearer for someone reading the code.